apps/passport-client/components/shared/PCDCardList.tsx

```diff
@@ -0,0 +1,279 @@
+import { PCD } from "@pcd/pcd-types";
+import { SemaphoreIdentityPCDTypeName } from "@pcd/semaphore-identity-pcd";
+import _ from "lodash";
+import { useCallback, useMemo, useState } from "react";
+import styled from "styled-components";
+import { usePCDCollection } from "../../src/appHooks";
+import { PCDCard } from "./PCDCard";
+
+type Sortable = {
```

Comment:
may be worth making this a generic type, such that the `pcd` field below is morphed into a `value` field defined as `value: T` in this `Sortable` type.